src/sdk/cohort.ts

```diff
@@ -0,0 +1,79 @@
+import { PublicKey } from '@nucypher/nucypher-core';
+
+import { Porter } from '../characters/porter';
+import { ChecksumAddress } from '../types';
+
+export type Ursula = {
+  readonly checksumAddress: ChecksumAddress;
+  readonly uri: string;
+  readonly encryptingKey: PublicKey;
+};
+interface CohortParameters {
```

Comment:
I like to use `type` instead of `interface` to indicate that a type is "just a container" for data, and not a "contract" that should be implemented by classes.

```suggestion
type CohortParameters {
```